src/assets/i18n/de_DE.json

```diff
@@ -1038,6 +1038,13 @@
   "product.short_description.label": "Kurzbeschreibung",
   "product.specific_attributes.label": "Spezifische Attribute",
   "product.variations.text": "Variationen",
+  "product.warranty.code.text": "Garantie-Code:",
+  "product.warranty.detail.text": "Details",
+  "product.warranty.expire_date.text": "Läuft aus am:",
+  "product.warranty.heading.text": "Schützen Sie Ihren Einkauf:",
+  "product.warranty.no_protection_plan": "Kein Schutzplan",
```

Comment:
"Schutzplan" sounds quite dramatic. Is this the correct term to use in context of product warranties? Browsing the net, I'm not so sure about it.